libs/ng-morph/ng/bootstrap/get-bootstrap-fn.ts

```diff
@@ -1,23 +1,20 @@
-import { CallExpression, Node } from 'ts-morph';
+import { CallExpression, Node, SyntaxKind } from 'ts-morph';
 import { getImports } from 'ng-morph/imports';
-import { getActiveProject } from 'ng-morph/project';
 
-export function getBootstrapFn(mainFilePath: string): CallExpression {
+export function getBootstrapFn(mainFilePath: string): CallExpression | undefined {
```

Comment:
I see it as a breking change because it can cause TS errors in old scripts after update

Reply:
If strict mode is enabled for schematics then probably yes it's a breaking change, but I believe it's how it should work after standalone apps release